components/raftstore/src/store/memory.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use lazy_static::lazy_static;
+use memory_trace_macros::MemoryTraceHelper;
+use std::sync::Arc;
+use tikv_alloc::{
+    mem_trace,
+    trace::{Id, MemoryTrace, MemoryTraceNode},
+};
+
+lazy_static! {
+    pub static ref MEMTRACE_ROOT: Arc<MemoryTraceNode> =
+        mem_trace!(raftstore, [peers, applys, raft_context, apply_context]);
+    pub static ref MEMTRACE_PEERS: Arc<dyn MemoryTrace + Send + Sync> =
+        MEMTRACE_ROOT.sub_trace(Id::Name("peers"));
+    pub static ref MEMTRACE_APPLYS: Arc<dyn MemoryTrace + Send + Sync> =
+        MEMTRACE_ROOT.sub_trace(Id::Name("applys"));
+    pub static ref MEMTRACE_RAFT_CONTEXT: Arc<dyn MemoryTrace + Send + Sync> =
+        MEMTRACE_ROOT.sub_trace(Id::Name("raft_context"));
+    pub static ref MEMTRACE_APPLY_CONTEXT: Arc<dyn MemoryTrace + Send + Sync> =
+        MEMTRACE_ROOT.sub_trace(Id::Name("apply_context"));
+}
+
+#[derive(MemoryTraceHelper, Default)]
+pub struct PeerMemoryTrace {
```

Comment:
`RouterTrace` is defined in router mod, should we keep the consistent?

Reply:
🤔 I think it would be better to define `RouterTrace` and `Router` in the same place.